src/views/EventViews001.js

```diff
@@ -1,278 +0,0 @@
-/**
```

Comment:
Can you add a comment somewhere about the server sent events? That way if we do need to add them back in it's easy to know what the issue is.